common/src/main/java/com/microsoft/identity/common/internal/platform/DevicePopManager.java

```diff
@@ -562,7 +564,7 @@ public boolean verify(@NonNull final SigningAlgorithm alg,
         try {
             final KeyStore.PrivateKeyEntry keyEntry = mKeyManager.getEntry();
 
-            if (!(keyEntry instanceof KeyStore.PrivateKeyEntry)) {
+            if (keyEntry == null) {
```

Comment:
Not clear to me why this check is changing - do we no longer need to test that its an instanceof KeyStore.PrivateKeyEntry even if != null?

Reply:
It's a redundant check (as in, mKeyManager's type is IKeyManager<KeyStore.PrivateKeyEntry>)

Reply:
Is it possible that it would return something else that is not inherited from IKeyManager<KeyStore.PrivateKeyEntry>? Not sure why it was added in the first place.

I got rid of it because of AS showed me a warning.

Reply:
Well... actually.. if it's a type erasure, it would have crashed in 563.